src/tests/conformance/post_handshake/validators.rs

```diff
@@ -257,14 +255,39 @@ async fn c026_TM_VALIDATOR_LIST_send_validator_list() {
         signature,
         version: 1,
     });
-    synth_node
+    synth_node1
         .unicast(node.addr(), payload)
         .expect("unable to send message");
 
-    // TODO: confirm result from rippled that the message was valid
-    // will be done in new PR
+    let start = Instant::now();
+    let check = |m: &BinaryMessage| {
+        let elapsed = start.elapsed();
+        let millis = elapsed.as_millis();
+        if millis > TIMEOUT_MILLIS {
+            assert!(false);
+        }
+        if let Payload::TmValidatorListCollection(validator_list_collection) = &m.payload {
+            if let Some(blob_info) = validator_list_collection.blobs.first() {
+                let decoded_blob =
+                    base64::decode(&blob_info.blob).expect("unable to decode a blob");
 
-    sleep(Duration::from_secs(5)).await;
-    synth_node.shut_down().await;
-    node.stop().expect("unable to stop stateful node");
+                let text = String::from_utf8(decoded_blob)
+                    .expect("unable to convert decoded blob bytes to a string");
+
+                let validator_list = serde_json::from_str::<ValidatorList>(&text)
+                    .expect("unable to deserialize a validator list");
+
+                // Only our message has a single validator, so we skip the others
+                if validator_list.validators.len() == 1 {
+                    assert!(validator_list.sequence == RAND_SEQUENCE_NUMBER);
+                    assert!(validator_list.validators[0]
+                        .validation_public_key
+                        .eq(MASTER_PUBLIC));
+                    return true;
+                }
```

Comment:
this change I chose not to do:  I like the nesting, because it's happening at many levels.  The indents aid (IMHO) understanding at a glance.  The situations where I do do what you suggest, is when there is a huge amount of code within the if block, and we're not normally in the parsing pattern.

Reply:
Okay, but do consider this change:
```
                assert_eq!(validator_list.sequence, RAND_SEQUENCE_NUMBER);
                assert_eq!(validator_list.validators[0].validation_public_key, MASTER_PUBLIC);
```

I should have done this in another comment, sorry about that. Because using `eq()` directly feels strange... tbh